src/lib/components/composites/project-components/ProjectListEntry.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+    import { Progress } from "$lib/components/primitives/progress";
+    import { goto } from "$app/navigation";
+    import type { Project, User } from "$lib/model/backend";
+
+    interface ProjectListEntryProps {
+        project: Project;
+        members: User[];
+        stage: number;
+        stageProgress: number;
+    }
+
+    const { project, members, stage, stageProgress }: ProjectListEntryProps = $props();
+
+    /**
+     * Combine the first- and lastname of the members array into a string as following:
+     * [{firstName: "John", lastName: "Doe", email: "john@doe.com"}, ... ] -->
+     * "John Doe, ..."
+     */
+    const getMemberNames = () => {
+        return members.map((member: User) => `${member.firstName} ${member.lastName}`).join(", ");
+    };
+</script>
+
+<!--
+@component
+Container displaying important project information.
+
+This component shows the
+  - project id and name
+  - members of the project
+  - current stage
+  - current stage progress (as progress bar, whereas the parameter must be provided as percentage)
+
+Furthermore this component is clickable and navigates to the corresponding project homepage.
+
+Usage:
+```svelte
+    <ProjectListEntry project={demoProject} members={memberUserSpecArray} stage={3} stageProgress={70} />
+```
+-->
+<button
+    type="button"
+    class="flex flex-col lg:flex-row h-fit w-full gap-3 lg:gap-10 items-start lg:items-center justify-between px-5 py-2
```

Comment:
`gap-3` seems a lot to me. What do you think about `gap-2` or `gap-1.5`?
```suggestion
    class="flex flex-col lg:flex-row h-fit w-full gap-2 lg:gap-10 items-start lg:items-center justify-between px-5 py-2
```
```suggestion
    class="flex flex-col lg:flex-row h-fit w-full gap-1.5 lg:gap-10 items-start lg:items-center justify-between px-5 py-2
```

Reply:
I can agree on 2, but 1.5 is too tight imo. What do you think of the change?

Reply:
That's fine!